railties/lib/rails/engine.rb

```diff
@@ -615,6 +615,15 @@ def load_seed
       end
     end
 
+    initializer :add_fixture_paths do
+      next if is_a?(Rails::Application)
+
+      fixtures = config.root.join("test", "fixtures")
```

Comment:
This should get from `config.paths`. It means we need to register this path, but we should not try to use the `root` because it disallow people from configuring this path.

Reply:
I looked into `paths` but wasn't really sure if it fit the use case here.